ARCs/arc-0004.md

```diff
@@ -266,10 +266,32 @@ from method invocations.
 
 The primary purpose of bare Application calls is to allow the
 execution of an OnCompletion (`apan`) action which requires no
-inputs and has no return value. A Contract **MAY** allow this
-for all OnCompletion actions, for only a subset of them, or for
-none at all. Great care should be taken when allowing these
-operations.
+inputs and has no return value. A Contract **MAY** allow this for
+all of the OnCompletion actions listed below, for only a subset of
+them, or for none at all. Great care should be taken when allowing
+these operations.
+
+Allowed OnCompletion actions:
+* 0: NoOp
+* 1: OptIn
+* 2: CloseOut
+* 4: UpdateApplication
+* 5: DeleteApplication
+
+Note that OnCompletion action 3, ClearState, is **NOT** allowed to
+be invoked as a bare Application call.
+
+> While ClearState is a valid OnCompletion action, its behavior differs
+> significantly from the other actions. Namely, an Application running during
+> ClearState which wishes to have any effect on the state of the chain
```

Comment:
"an application running during"

Reply:
Is the objection to the capital A? If so it was decided some time ago that Application should be capitalized in this doc

Reply:
more that it is awkwardly phrased and you are describing something very delicate.